### DISCUSSION TITLE
mariadb: unrecognized service

### DISCUSSION THREAD

**Discussion by DrRavik90** [monarc-project/MonarcAppFO#503] [Q&A]
Hi there,
Since i was having trouble and i was a lot confused (even here it could be a lot of help for any video or manual or script) installing MONARC in UBUNTU i tried into WSL.

It's been a lot that it is stuck into: mariadb: unrecognized service

I do have: Error for configuring php ( sed: can't read /etc/php/8.1/apache2/php.ini: No such file or directory

And in fact there is no such directory.

I do have: Error Configuring Xdebug for development part> bash: /etc/php/8.1/..... No such file or directory

I do have Error for restarting Apache

Error PHP Parse

Error Restarting Apache

What is going on here ? Is this script updated or checked ? 

Can anyone help ?

Thank You

**Comment by ruslanbaidan** [comment]
Hello,
Do you plan to install Monarc to perform some development for the project ?
as WSL is for this reasons: `[WSL](https://docs.microsoft.com/en-us/windows/wsl/) is convenient to use in order to setup your development environment on Windows OS.`
If you would like to use Monarc locally for testing purposes or even production you could use VM solution:
https://www.monarc.lu/download/#virtual-machine
The latest VM is available here: https://vm.monarc.lu/latest/

**Reply by DrRavik90** [reply]
I want it for production reasons. Can i use even the VM ?

**Reply by ruslanbaidan** [reply]
yes, it is used for production, but the version updates could be complicated to do, DB backups have to be done manually.
The best solution, you could install it on your linux server/machine or request a SaSS solution for your organisation:
https://www.monarc.lu/service/

**Reply by DrRavik90** [reply]
I understand. I'm trying to install it on Ubuntu 20.04.6 but no luck till now :(

I followed the steps mentioned here: https://github.com/monarc-project/MonarcAppFO/blob/master/INSTALL/INSTALL.ubuntu2004.md

From 1-1.3 it is okej. However i noticed that some part of the steps in https://github.com/monarc-project/MonarcAppFO/blob/master/INSTALL/INSTALL.ubuntu2004.md are not the same as here https://github.com/monarc-project/MonarcAppFO/blob/master/INSTALL/INSTALL.ubuntu1804.md and part of the steps are even skipped. 

For example; Step 1.4 is not full described for 20.4. 

Is there any script to automate the whole installation on ubuntu ? 

I installed DB but i cannot do the Initialization of  the database. I also believe the script is not okej:

$ mysql -u monarc -ppassword monarc_common < db-bootstrap/monarc_structure.sql
$ mysql -u monarc -ppassword monarc_common < db-bootstrap/monarc_data.sql

I believe it must be a space at the -p password part ? What is password ? what if i have changed the password on the step above ?

anyway i have already tried with and without space and it does not happen anything . 

Another error: 
administrator@rmaserver:~$ sudo cp ./config/autoload/local.php.dist ./config/autoload/local.php
[sudo] password for administrator:
cp: cannot stat './config/autoload/local.php.dist': No such file or directory

**Reply by ruslanbaidan** [reply]
You need to be in the project root directory as the files' paths are relative.

**Reply by DrRavik90** [reply]
Pls: what about i use it as VM it means that i can use it only locally. Is there any way to use it as vm for my colleague
 to access it too ?

**Reply by ruslanbaidan** [reply]
The question is out of the scope of Monarc use or configuration.
For example, configuring local machine as a server could be a solution.
